_tuning-your-cluster/performance.md

```diff
@@ -134,3 +134,7 @@ curl -XPOST "es-endpoint/index-name/type-name/_bulk?pretty&filter_path=-took,-it
 { "doc" : {"user" : "example"} }
 ```
 {% include copy.html %}
+
+## Compression codecs
+
+Starting with OpenSearch 2.9, new codecs for compression are made available: `zstd` and `zstd_no_dict`. You can optionally specify a compression level for these in the `index.codec.compression_level` setting with values in the range [1,6]. These provide improved indexing performance at the expense of a slightly larger index size. [Benchmark]({{site.url}}{{site.baseurl}}/im-plugin/index-codecs/#benchmarking) data shows zstd providing 7% better writes throughput, and zstd_no_dict providing 14% better throughput. You can find more details about compression on [Indexing codecs]({{site.url}}{{site.baseurl}}/im-plugin/index-codecs).
```

Comment:
```These provide improved indexing performance at the expense of a slightly larger index size.```
this is slightly incorrect, we can remove this line.

shows ```zstd``` providing 7% better writes throughput, and ```zstd_no_dict``` providing 14% better throughput alongside over 30% improvements in storage over current ```default``` codec.

you can say something like this, feel free to rephrase

Reply:
I can edit it. However, the https://opensearch.org/docs/latest/im-plugin/index-codecs/ does mention the storage tradeoff.

Reply:
the storage trade off mentioned is between ```zstd``` and ```zstd_no_dict```. The indices created with ```zstd_no_dict``` will have slightly larger indices than those created with ```zstd```
In your previous revision, it seemed like there is a trade off with respect to current lz4 (default codec)

Reply:
This LGTM now! Thanks @akashsha1